datafusion/sql/src/values.rs

```diff
@@ -31,8 +33,9 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             rows,
         } = values;
 
-        // Values should not be based on any other schema
-        let schema = DFSchema::empty();
+        let schema = planner_context
+            .table_schema()
+            .unwrap_or(Arc::new(DFSchema::empty()));
```

Comment:
```suggestion
        let schema = planner_context
            .table_schema();
```

and use `Option` below

Reply:
schema is used in `self.sql_to_expr(v, &schema, planner_context)` too, so I think it is better to have empty schema if table schema is not exist

Reply:
That's good point. I think `sql_to_expr` must still get empty schema. -- The VALUES being constructed cannot refer to columns of the table being created.